app/views/administrate/application/edit.html.erb

```diff
@@ -15,13 +15,13 @@ It displays a header, and renders the `_form` partial to do the heavy lifting.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Form
 %>
 
-<% content_for(:title) { "Edit #{page.page_title}" } %>
+<% content_for(:title) { "#{t("administrate.actions.edit")} #{page.page_title}" } %>
```

Comment:
What about passing the page title as an option to the translation? That would be good for languages where the title makes more sense before the edit word. What do you think?